core/src/test/java/org/elasticsearch/explain/ExplainActionTests.java

```diff
@@ -138,7 +139,7 @@ public void testExplainWithFields() throws Exception {
         assertThat(response.getExplanation().getValue(), equalTo(1.0f));
         assertThat(response.getGetResult().isExists(), equalTo(true));
         assertThat(response.getGetResult().getId(), equalTo("1"));
-        assertThat(response.getGetResult().getFields().size(), equalTo(1));
+        assertThat(response.getGetResult().getFields().size(), greaterThanOrEqualTo(1));
```

Comment:
I could change the assertion to something like `fields.keySet().containsAll(Arrays.asList(expectedField1, expectedField2))`?